config/bundles.php

```diff
@@ -13,7 +13,6 @@
     Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
     Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
```

Comment:
We aren't using Nelmio Bundle (Api Doc) as well, should be removed

Reply:
I think that's how /api creates the doc?

Reply:
Yes, I thought no one uses it? and we are on an older version as well

Reply:
We use it. And we can upgrade to 3.0, but maybe not now. Would rather put effort into upgrading to API Platform.